### ISSUE TITLE
Using workflow as pattern (Workflow Management System)

### ISSUE THREAD

**Issue by fatroze2000** [aelassas/Wexflow#8]
Hi
Already Wexflow run some tasks in a specific pattern.
How can I define a workflow (pattern) which get a object as input and run some instances of workflows in run time? Maybe at a moment 2 versions of workflow in first task, 3 versions of workflow in second task and etc.
In other hand, I need Workflow Management System like sharepoint.
As example I need a workflow for loaning or registering in gym.

**Comment by aelassas** [comment]
hi,
If I create a task that takes as input a list of workfow ids and then launches these workflows. Would that be a solution?
Regards,

**Comment by fatroze2000** [comment]
hi
like bpms, it should be run some instances of  some workflows.

-------- پیام اصلی --------
از: Akram EL ASSAS <notifications@github.com> 
تاریخ:۰۵/۱۸/۲۰۱۷  ۲۱:۳۷  (GMT+03:30) 
گیرنده: aelassas/Wexflow <Wexflow@noreply.github.com> 
گیرنده کپی: Yaser Yazdanipour <yasser_yazdani@yahoo.com>, Author <author@noreply.github.com> 
موضوع: Re: [aelassas/Wexflow] Using workflow as pattern (Workflow Management
  System) (#8) 

hi, If I create a task that takes as input a list of workfow ids and then launches these workflows. Would that be a solution? Regards, —You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by aelassas** [comment]
Ok, I'm working on it.

**Comment by aelassas** [comment]
Ok, It's done.

I've created a new task named Workflow that allows to start, suspend, resume or stop a list of workflows. It is available in the new release [v1.1](https://github.com/aelassas/Wexflow/releases/tag/v1.1).

Uninstall Wexflow then install the new release through the windows installer then you're done.

Here is a sample workflow that uses the task:

```xml
<?xml version='1.0'?>
<Workflow xmlns="urn:wexflow-schema" id="45" name="Workflow_Workflow" description="Workflow_Workflow">
	<Settings>
		<Setting name="launchType" value="trigger" /> <!-- startup|trigger|periodic -->
		<Setting name="enabled" value="true" /> <!-- true|false -->
	</Settings>
	<Tasks>
		<Task id="1" name="Workflow" description="Starting workflow 43" enabled="true">
			<Setting name="wexflowWebServiceUri" value="http://localhost:8000/wexflow/" />
			<Setting name="action" value="start" /> <!-- start|suspend|resume|stop -->
			<Setting name="id" value="43" />
		</Task>
		<Task id="2" name="Workflow" description="Starting workflow 10 and 11" enabled="true">
			<Setting name="wexflowWebServiceUri" value="http://localhost:8000/wexflow/" />
			<Setting name="action" value="start" /> <!-- start|suspend|resume|stop -->
			<Setting name="id" value="10" />
			<Setting name="id" value="11" />
		</Task>
	</Tasks>
</Workflow>
```
This workflow launches the workflow 43 then the workflows 10 and 11.
